lib/livebook_web/helpers/html_helpers.ex

```diff
@@ -36,7 +36,12 @@ defmodule LivebookWeb.HTMLHelpers do
     name
     |> String.trim()
     |> String.downcase()
+    |> String.replace(~r/[\p{P}\p{S}]/, "")
```

Comment:
Instead of specifying which characters to replace, it's more reliable to all except those that we want to keep:

```suggestion
    |> String.replace(~r/[^\s\w]/u, "")
```

Reply:
Even with that, the emoji still breaks the regex

Reply:
Do you use the `u` modifier?

```elixir
iex(1)> "🐈" |> String.replace(~r/[^\s\w]/u, "")
""
```

Reply:
Hmmm, we're removing it but the test expects to have the emoji